docs/blockchain/proof-of-coverage.mdx

```diff
@@ -40,7 +40,7 @@ provided by Hotspots on the Network.
   [block 1,731,335](https://explorer.helium.com/blocks/1731335).
 - The first Oracled Proof of Coverage Reciepts were in
   [block 1,731,339](https://explorer.helium.com/blocks/1731339).
-- [Bitter Concrete Parakeet](https://explorer.helium.com/validators/12xtDibQdaAsvbPTqFbecGaA4iHd7NzRg6s7foQ2HCxouHo11TE)
+- [Bitter Concrete Parakeet](https://explorer-old.helium.com/validators/12xtDibQdaAsvbPTqFbecGaA4iHd7NzRg6s7foQ2HCxouHo11TE)
```

Comment:
Should we do a sweep on all explorer links? Would need this for the blocks above too.